docs/development/workflow/command_history.sh

```diff
@@ -22,20 +22,19 @@ git push --set-upstream mwcraig i-1761 # Hey GitHub, get ready to receive my wor
 #
 conda create -n apy-1761 --clone root   # copy my default environment
 source activate apy-1761                # switch to this environment
-# next step DOES NOT WORK in python 3
-python setup.py develop                 # install astropy
+pip install -e .                        # install astropy
 #
 # Check setup by running tests
 #
 cd astropy/coordinates/tests    # get ready to run coordinate tests
-py.test                         # test
+pytest                          # test
```

Comment:
Instead of these two lines, ``pytest astropy/coordinates`` would be better because it will load the pytest config in setup.cfg and will also run doctests not in the tests directory.